sleepecg/io/gudb.py

```diff
@@ -5,7 +5,7 @@
 """Utilities for downloading GUDB data."""
 
 from pathlib import Path
-from typing import Optional, Union
+from typing import Dict, Optional, Union
```

Comment:
Could we use the normal `dict` as a type hint? I know that this requires Python 3.9, but would it be preferable to already to it now with `from __future__ import annotations`?

Reply:
Oh right, good point!